src/System.Reflection.MetadataLoadContext/src/System/Reflection/TypeLoading/Types/Ecma/EcmaGenericTypeParameterType.cs

```diff
@@ -21,7 +21,6 @@ internal EcmaGenericTypeParameterType(GenericParameterHandle handle, EcmaModule
 
         protected sealed override RoType ComputeDeclaringType()
         {
-            MetadataReader reader = Reader;
```

Comment:
This also seems like a mistake, though in theory accessing Reader could have some kind of side effect?